undertow/src/main/java/org/wildfly/extension/undertow/security/LogoutNotificationReceiver.java

```diff
@@ -41,22 +40,34 @@
 public class LogoutNotificationReceiver implements NotificationReceiver {
 
     private final AuthenticationManager manager;
+    private final String securityDomain;
 
-    public LogoutNotificationReceiver(AuthenticationManager manager) {
+    public LogoutNotificationReceiver(AuthenticationManager manager, String securityDomain) {
```

Comment:
you might not need this - iirc you can retrieve the security domain from the current security context before clearing it